plugins/scroll-options/src/index.js

```diff
@@ -77,16 +77,9 @@ export class ScrollOptions {
       return;
     }
 
-    let element;
-    const dragSurface = this.workspace_.getBlockDragSurface();
-    if (dragSurface) {
-      element = dragSurface.getSvgRoot();
-    } else {
-      element = this.workspace_.svgGroup_;
-    }
-
+    // TODO: We should maybe add an accessor for the svgGroup_?
```

Comment:
Please open a bug about this (maybe with an explanation more comprehensible to the casual observer) and reference it here.  (Make the bug title along the lines of "consider whether to…" if you're not sure if we want to do this.)

Reply:
Done! https://github.com/google/blockly/issues/7157